File: ts/wam/2216.ts
export const WamBusinessToolsEntry = 2216 as const

export interface BusinessToolsEntryEvent {
  /** field 1, wire `business_tools_session_id` */
  businessToolsSessionId?: string
  /** field 2, wire `business_tools_sequence_number` */
  businessToolsSequenceNumber?: number
  /** field 3, wire `business_tools_entry_point` */
  businessToolsEntryPoint?: BUSINESSTOOLSENTRYPOINTTYPE
}

File: go/wam/2216.go
const WamBusinessToolsEntry = 2216 // channel: regular

type BusinessToolsEntryEvent struct {
	BusinessToolsSessionId string `wam:"business_tools_session_id"` // field 1
	BusinessToolsSequenceNumber int64 `wam:"business_tools_sequence_number"` // field 2
	BusinessToolsEntryPoint BUSINESSTOOLSENTRYPOINTTYPE `wam:"business_tools_entry_point"` // field 3
}